distribution/lib/Standard/Base/0.0.0-dev/src/Runtime.enso

```diff
@@ -22,9 +22,10 @@ get_stack_trace : Vector.Vector Stack_Trace_Element
 get_stack_trace =
     prim_stack = primitive_get_stack_trace
     stack_with_prims = Vector.from_polyglot_array prim_stack
-    stack = stack_with_prims.map wrap_primitive_stack_trace_element
-    # drop this frame and the one from `Runtime.primitive_get_stack_trace`
-    stack.drop (First 2)
+    # (First 2) drops the `Runtime.primitive_get_stack_trace` frame and this one
+    # (Last 1) drops the `org.graalvm.polyglot.Value<Function>.execute` frame
+    stack = stack_with_prims.drop (First 2) . drop (Last 1)
```

Comment:
`Runtime.primitive_get_stack_trace` returns internal `Value<Function>.execute` method as a top frame. It causes issues when this frame is passed to methods like `Polyglot.get_executable_name`. 
I.e the program
```py
Polyglot.get_executable_name (Runtime.primitive_get_stack_trace . at 2)
```
Returns `org.graalvm.polyglot.Value<Function>.execute` in CLI mode and fails with `Null receiver values are not supported by libraries.` in interactive (IDE) mode.

I wonder, if it would be a better solution to update the `Runtime.primitive_get_stack_trace` builtin method so that it doesn't return this internal frame?